### PR TITLE
Some Changes

### PR DESCRIPTION
**Pull request by sijad**
This tool is very useful for me, with this changes it's easier to use this

### CODE REVIEW

**Comment by mattn** [comment]
merged.

File: src/extension.ts [hunk 1]
```diff
@@ -74,24 +75,29 @@ export function activate(ctx: vscode.ExtensionContext): void {
...
+      fileName = path.relative(cwd, fileName);  
     var output = vscode.window.createOutputChannel('Runner: ' + action + ' ' + fileName);
-    output.show();
+    output.show(vscode.ViewColumn.Two);
+    editor.show()
     var sh = win32 ? 'cmd' : '/bin/sh';
     var shflag = win32 ? '/c' : '-c';
     var args = document.isDirty || document.isUntitled ? [shflag, action] : [shflag, action + ' ' + fileName];
     var child = cp.spawn(sh, args, { cwd: cwd, detached: false });
+    var clearPreviousOutput = vscode.workspace.getConfiguration('runner')['clearPreviousOutput'] || true;
```
Comment (openingnow) [inline src/extension.ts:87]: Isn't the `true` should be `false`?

**Comment by mattn** [comment]
@sijad do you have opinion?

File: src/extension.ts (hunk 1 above)
Comment (mattn) [inline src/extension.ts:87]: @sijad do you have opinion?

Comment (sijad) [inline src/extension.ts:87]: ~~as the [default value](https://github.com/mattn/vscode-runner/blob/bce25b04ace32e66b0e919571a68ed1f14ad936e/package.json#L60) is `true` I think `true` is fine~~
i can see his point, i guess both of them should be false, sorry about that, PR is comming

**Pull request closed without merging**